src/components/Vaults/index.tsx

```diff
@@ -26,8 +27,8 @@ const Vaults = ({ label, requiredCapacity, isShown, onSelectionCallback, error }
     // Filters out vaults with lower than required capacity and sorts by accountId
     // to have vaults with same accountId grouped together.
     const vaultsWithEnoughCapacity = allVaults
-      .filter((vault) => vault[1].gt(BitcoinAmount.from.Satoshi(0)))
-      .filter((vault) => vault[1].gte(BitcoinAmount.from.Satoshi(requiredCapacity)))
+      .filter((vault) => vault[1].gt(BitcoinAmount.zero())) // TODO: redundant check
```

Comment:
What is the list of vaults used for?
I'm wondering if the vault list is even needed if `requiredCapacity` is 0 or `undefined`.

Maybe an early exit if `requiredCapacity || 0 <= 0` (return empty array or all vaults), and just a `.gt(requireCapacity)` filter otherwise is an option?

Reply:
@peterslany ^

Reply:
@bvotteler the list is used in manual vault selection component on Issue page when user wants to select the vault manually - therefore we should display the vaults even if the capacity is `0` or `undefined` to let user have an option to first select a vault and then input the amount to issue (which in this case is `requiredCapacity`)